### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by DStatPersonal**

### CODE REVIEW

File: adding_a_license_to_your_repository_v2.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+---
+slug: Adding-a-License-to-Your-Repository
```
Comment (prokopsimek) [inline adding_a_license_to_your_repository_v2.md]: please use small letter in all words

File: adding_a_license_to_your_repository_v2.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
+---
+slug: Adding-a-License-to-Your-Repository
+aspect: processes
```
Comment (prokopsimek) [inline adding_a_license_to_your_repository_v2.md]: "aspect" is deprecated, remove it please

File: adding_a_license_to_your_repository_v2.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+keywords:
+  - license
+  - repository
+  - royalty terms
+  - copyright
+---
+
+# Adding a License to Your Repository
+
+## TL;DR
```
Comment (prokopsimek) [inline adding_a_license_to_your_repository_v2.md]:
```suggestion
**TL;DR**
```

use just bold for tl;dr pls

File: adding_a_license_to_your_repository_v2.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+  - installation
+  - writing
+keywords:
+  - license
+  - repository
+  - royalty terms
+  - copyright
+---
+
+# Adding a License to Your Repository
```
Comment (prokopsimek) [inline adding_a_license_to_your_repository_v2.md]: I´m not sure about that long name. Could we call it just e.g. "License in Repository" or something similar?

File: adding_a_license_to_your_repository_v2.md (hunk 2 above)
Comment (prokopsimek) [reply adding_a_license_to_your_repository_v2.md]:
```suggestion
```

File: adding_a_license_to_your_repository_v2.md (hunk 1 above)
Comment (prokopsimek) [reply adding_a_license_to_your_repository_v2.md]:
```suggestion
slug: adding-a-license-to-your-repository
```

File: adding_a_license_to_your_repository_v2.md (hunk 3 above)
Comment (Kristynaslomkova) [reply adding_a_license_to_your_repository_v2.md]: @prokopsimek I can’t accept your suggestions because I don’t have permission to push to this repository
